tuning/autotune.py

```diff
@@ -185,6 +196,11 @@ def get_benchmark_time(self) -> Optional[float]:
         except ValueError:
             return None
 
+    def generate_sample_result(
+        self, candidate_id: int = 0, mean_time: int | float = random.randint(100, 500)
```

Comment:
```suggestion
        self, candidate_id: int = 0, mean_time: float = random.uniform(100.0, 500.0)
```
Make this always be a float?
Also, what's the unit of time here?